glance/manifest.jsonnet

```diff
@@ -81,6 +81,13 @@ kpm.package({
       glance_keyring: "",
     },
 
+    swift: {
+      enabled: false,
+      auth_address: "http://10.91.62.52:80/auth/1.0",
```

Comment:
we should change it to radosgw/auth/1.0 and have radosgw deployed on kubernetes. Some time ago we had a radosgw in stackanetes so we may reuse it.